portfolio/JSONFile/Spring21-5b3114d73b42.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Please remove this file and regenerate your API key. Because this file is now public, it is possible for third parties to use your API account.

After you regenerate this file, you can add it to your .gitignore file so that it isn't uploaded to GitHub.

Reply:
How do I regenerate the API  key?

Reply:
You can do so from the credentials page: https://console.cloud.google.com/apis/credentials?pli=1

Click on your service account, go to Keys, remove the existing key, and create a new one.

Reply:
Okay thank you, Ruiqi.